tsl/test/sql/chunk_utils_internal.sql

```diff
@@ -394,6 +394,14 @@ FROM chunk_view
 WHERE hypertable_name = 'hyper_constr'
 ORDER BY chunk_name;
 
+--Utility functions -check index creation on hypertable with OSM chunk 
```

Comment:
IMHO will be nice if you can improve a bit the comments here for people understand that we're skipping CREATE INDEX on OSM chunks...

Reply:
added more comments here.